src/cogs/mirror/mirror.py

```diff
@@ -1,35 +1,38 @@
+from linecache import cache
 import discord
 from discord.ext import commands
 from discord.ext.commands import Context
 
 
 class Mirror(commands.Cog):
 
-    def __init__(self, bot, cache: dict):
+    def __init__(self, bot):
         self.bot = bot
-        self.cache: dict = cache
```

Comment:
reee just self.cache: dict = bot.cache

Reply:
this is not a great way to do code reviews

Reply:
reeeeeeee

Reply:
i mean i guess another option, since we're passing around self, is to just use self.cache in the bot factory, and then we don't have to assign it here.  but i think i still might find that a little confusing in the individual methods.  i would typically think self is particular to an instance or a class, but that's not how it's working here.